packages/types-terra/src/interfaces.ts

```diff
@@ -50,3 +50,9 @@ export interface TerraEvent<M extends Msg = Msg> {
 export type TerraContractMessage = TerraMessage<WasmMsg>;
 
 export type DynamicDatasourceCreator = (name: string, args: Record<string, unknown>) => Promise<void>;
+
+export interface ITerraSafeApi {
```

Comment:
We want this api to match LCD client, but partially implemented. Currently these methods are a layer too high. We want it to be `api.wasm.contractInfo` but it is currently `api.contractInfo`. 

Follow the terra repo for more https://github.com/terra-money/terra.js/blob/main/src/client/lcd/LCDClient.ts#L100

This should look something like:
```ts
export interface SafeLCD {

    wasm: typeof WasmAPI;
}

```